### PR TITLE
added changes to team section and more response on a smaller devices

### PR DESCRIPTION
**Pull request by Decomplosed**
take a look at those changes

### CODE REVIEW

File: assets/style/style.css [hunk 1]
```diff
@@ -636,24 +636,18 @@ h2 {
...
+    margin: 20px 40px;
     text-align: center;
-    margin-bottom: 50px;
-    margin-top: 50px;
-    padding-top: 50px;
-    padding-bottom: 50px;
+    padding-top: 20px;
+    padding-bottom: 20px;
     filter: grayscale(80%);
 }
```
Comment (icelandico) [inline assets/style/style.css:650]: Maybe remove these wide whitespaces between selectors.

File: assets/style/style.css [hunk 2]
```diff
@@ -724,8 +718,17 @@ h2 {
...
+        width: 60%;
+    }
+
+    .team-photo{
+    margin-bottom: 5px;
     }
 }
+
+
+@media (max-width:576px) {
```
Comment (icelandico) [inline assets/style/style.css:734]: If there's nothing more in this `media` query, it might be deleted.

File: assets/style/style.css [hunk 3]
```diff
@@ -714,6 +702,12 @@ h2 {
 }
 
 @media (max-width:1024px) {
```
Comment (icelandico) [inline assets/style/style.css:704]: Maybe use `min-width: 968px` for consistency in our style.css

**Review by icelandico — CHANGES_REQUESTED** [review]
Just minor changes or rather things to consider.

File: assets/style/style.css [hunk 4]
```diff
@@ -484,7 +484,7 @@ h2 {
     opacity: 0.3;
 }
 
-.fa-facebook-square:hover, .fa-linkedin:hover, .fa-instagram:hover   {
+.fa-facebook-square:hover, .fa-linkedin:hover, .fa-instagram:hover, .fa-github-square:hover{
```
Comment (bartzaw) [inline assets/style/style.css:487]: no need for facebook & instagram icons, please remove them. Moreover there's already hover for linkedin Icon in footer section, so you are going to have conflicting styles in here.

File: assets/style/style.css [hunk 5]
```diff
@@ -636,24 +636,18 @@ h2 {
...
-    margin-left: 40px;
-    margin-right: 40px;
+    margin: 20px 40px;
     text-align: center;
-    margin-bottom: 50px;
-    margin-top: 50px;
-    padding-top: 50px;
-    padding-bottom: 50px;
+    padding-top: 20px;
+    padding-bottom: 20px;
```
Comment (bartzaw) [inline assets/style/style.css:647]: padding: 20px 0; is what you are looking for here. No need to make it in separate lines

File: assets/style/style.css [hunk 6]
```diff
@@ -671,28 +665,22 @@ h2 {
```
Comment (bartzaw) [inline assets/style/style.css:665]: All icons have the same margin, why all of them are separate? + margin: 0 14px;

**Review by bartzaw — CHANGES_REQUESTED** [review]
please check out my comments

File: assets/style/style.css (hunk 4 above)
Comment (Decomplosed) [reply assets/style/style.css:487]: facebook and instagram icons are removed from team-member, only added github to styles, so they would have exactly the same animation as icons in footer

Comment (bartzaw) [reply assets/style/style.css:487]: got it! we can discuss it today

**Comment by Decomplosed** [comment]
pull request closed

**Pull request closed without merging**